### ISSUE TITLE
[Not an issue] For those who still want first and last tab separator - Here's how

### ISSUE THREAD

**Issue by Minibus93** [QNetITQ/WaveFox#142]
Hello,

sorry for opening an issue that isn't really an issue, I just wanted to provide people a way to still have first and last tab separator in 114.
I know that first and last tab separator weren't liked apparently ( #139 ), but I did and maybe someone else, like me, did too.

I didn't want to bother asking you to revert it, so I did some digging and found a way that wanted to share with other people who may be here looking for a way to do it.

So, for those who are looking for the way, [here it is](https://github.com/QNetITQ/WaveFox/commit/8a05c7723753f470bb16f26091436a6c2c15ca3b)

Basically, this is the commit where QNetITQ made the change, you just need to download the new WaveFox and then remove the new lines and replace them with the old ones that are provided by the commit history.
After doing this, save it, and you'll have WaveFox 1.6.114 (FF114 Ready) with the old separator behavior!

Hope this may help people!

Thanks QNetITQ for the hard work as always!

**Comment by QNetITQ** [comment]
The current implementation of tab separators is the most optimal. To make it more flexible is not yet possible. I'm waiting for the browser to add full support for the `:has` pseudo-class. It has already been added, but not fully implemented. Without it, I can't make tab separators more flexible.
